deepmd/jax/modifier/dipole_charge.py

```
@@ -0,0 +1,171 @@
+            return ewald_reciprocal_energy(
+                jnp.concatenate(
+                    [strained_coord, selected_coord + selected_dipole], axis=1
+                ),
+                jnp.concatenate([real_charge, wc_charge], axis=1),
+                strained_box,
+                grid,
+                self.ewald_beta,
+            )
+
+        strain = jnp.zeros((coord.shape[0], 3, 3), dtype=coord.dtype)
+        energy, gradients = jax.value_and_grad(
```

Ruff (0.15.20)

[warning] Unpacked variable `energy` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)